enocean/__init__.py

```diff
@@ -526,6 +526,22 @@ def update_item(self, item, caller=None, source=None, dest=None):
                         self.logger.debug('enocean: item is A5_38_08_03 type')
                         self.send_dim(id_offset, item(), 0)
                         self.logger.debug('enocean: sent dim command')
+                    elif(tx_eep == 'D2_01_07'):
+                        if 'enocean_rx_id' in item.conf:
+                            rx_id = int(item.conf['enocean_rx_id'],16)
+                            self.logger.info('enocean:  enocean_rx_id found')
+                        else:
+                            rx_id=0
+                            self.logger.info('enocean:  NO enocean_rx_id found')
+                        if 'pulsewidth' in item.conf:
+                            pulsew = float(item.conf['pulsewidth'])
+                            self.logger.info('enocean:  pulsewidth found')
+                        else:
+                            pulsew=0
+                            self.logger.info('enocean:  NO pulsewidth found')
+                        self.logger.info('enocean: item is D2_01_07_01 type')
+                        self.send_switch_D2(id_offset, rx_id, pulsew, item())
+                        self.logger.debug('enocean: sent switch command for D2 VLD')
```

Comment:
Do you really want to log this stuff above using `INFO` level? I would prefer to use `DEBUG` instead.

Reply:
You're right, DEBUG is better. Could you please change it.

Reply:
I can't change it. Just change it in your local copy, then commit to branch and push it again. The PR will be updated then.

Reply:
I just changed the INFO into DEBUG. Furthermore, I renamed pulsewidth to enocean_pulsewidth so that we are more consistent with the names of the attributes.